util/libssl.num

```diff
@@ -497,3 +497,4 @@ SSL_get_recv_max_early_data             497	1_1_1	EXIST::FUNCTION:
 SSL_CTX_get_recv_max_early_data         498	1_1_1	EXIST::FUNCTION:
 SSL_CTX_set_recv_max_early_data         499	1_1_1	EXIST::FUNCTION:
 SSL_CTX_set_post_handshake_auth         500	1_1_1	EXIST::FUNCTION:
+SSL_get_signature_type_nid              501	1_1_1a	EXIST::FUNCTION:
```

Comment:
Is this being backported? If so this should be `1_1_0j`...although I'm not sure about the backport since 1.1.0 is in security fix only mode.

Reply:
You have a point about the symbol version, if backported, it would have to be a "1_1_0j" symbol, that would then be provided also in the the 1.1.1a library. So we should promptly decide whether to backport or not. On further reflection, so long as we're not removing the one macro that I renamed and aliased to the new name under the original, we may not need a backport, we just need to not abandon the legacy interface too quickly (if ever, the extra macro costs little).